src/content/protocols/liquity-v2/ethereum.md

```diff
@@ -0,0 +1,160 @@
+---
+chain: "Ethereum"
+stage: 2
+reasons: []
+risks: ["L", "L", "L", "L", "L"]
+author: ["Rick Pardoe"]
+submission_date: "2025-06-07"
+publish_date: ""
+stage_requirements: [
+      [
+            { text: "Assets are not in custody by a centralized entity", status: "fixed" },
+            { text: "All contracts are verified", status: "fixed" },
+            { text: "Source-available codebase", status: "fixed" },
+            { text: "Public documentation exists", status: "fixed" },
+      ], 
+      [
+            { text: "Upgrades have no potential of “loss of funds“ ", status: "fixed"},
+            { text: "Dependency with a High centralization score is mitigated", status: "fixed" },
+            { text: "Frontend backups or self-hosting option exists", status: "fixed"},
+      ], 
+      [
+            { text: "Contracts are immutable", status: "fixed" },
+            { text: "Dependency with a High or Medium centralization score is mitigated", status: "fixed" },
+            { text: "Alternative third-party frontends exist", status: "fixed" }  
+      ]]
+---
+
+# Summary
+
+Liquity v2 is a decentralized borrowing protocol that lets users deposit ETH or LSTs as collateral and mint the stablecoin BOLD. 
```

Comment:
We try not to use adjectives that may be subjective, like "decentralized", some more precise alternatives to describe BOLD could be suitable.

```suggestion
Liquity v2 is a borrowing protocol that lets users deposit ETH or Liquid Staking Tokens (LSTs) as collateral and mint the stablecoin `BOLD`. 
```